app/controllers/ops_controller/settings/label_tag_mapping.rb

```diff
@@ -192,7 +192,7 @@ def label_tag_mapping_add(entity, label_name, cat_description)
                                                    :description  => cat_description_with_prefix(entity, cat_description),
                                                    :single_value => true,
                                                    :read_only    => true)
-        ContainerLabelTagMapping.create!(:labeled_resource_type => entity,
+        ProviderTagMapping.create!(:labeled_resource_type => entity,
```

Comment:
maybe line up the arrows vertically as they were before your change

Reply:
:+1: done